orttraining/orttraining/python/training/api/module.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# module.py
+
+from onnxruntime.capi import _pybind_state as C
+from onnxruntime.capi.onnxruntime_inference_collection import OrtValue
+from onnxruntime.capi.onnxruntime_pybind11_state import OrtValueVector
+
+
+class Module:
+    """
+    Class for running Training.
+    This class is a wrapper of Module Class.
+    """
+
+    training: bool
+
+    def __init__(self, train_model_uri, state, eval_model_uri=None) -> None:
+        """
+        Initializes Model for Training.
+        __init__ will call an internatl function to create the model.
+        """
+        # TODO : Add support for bytes on train_model_uri and eval_model_uri.
+        self.training = True
+        self._model = C.Module(train_model_uri, state._state, eval_model_uri)
+
+    def __call__(self, user_inputs):
+        """
+        This method enables calling Module as a function to run the model.
+        Args:
+            user_inputs (OrtValueVector): input vector of ortvalues.
+        Returns:
+            fetches : OrtValueVector that has model output.
```

Comment:
Can user inputs and fetches both be python lists?

Reply:
I agree! Conversion to OrtValue can happen internally